app/controllers/users_controller.rb

```diff
@@ -47,4 +48,8 @@ def user_parameters
                                  :spire,
                                  :staff
   end
+
+  def allow_only_admin
+    deny_access && return unless @current_user.admin?
```

Comment:
[The application controller access control method](https://github.com/umts/jobapps/commit/489d6d9b72fc7705813da2ecce0f17fe3a72c152#diff-55c5b7aecfb519d0e4880eaf2788eb6eR21) uses `and return` rather than `&& return`.

They should be consistent.